rudder-relay/SPECS/rudder-relay.spec

```diff
@@ -118,20 +118,6 @@ Rudder is an open source configuration management and audit solution.
 This package is essentially a meta-package to install all components required to
 run a Rudder relay server on a machine.
 
-#=================================================
-# Source preparation
-#=================================================
-%prep
-%setup -c
-
-# We don't know the exact version
-cd rudder-sources-*/rudder/relay/sources/
```

Comment:
this is required

Reply:
what ? setup-c is required, but not the cd

Reply:
indeed